src/Components/OATPropertyEditor/PropertySelector.tsx

```diff
@@ -182,6 +182,13 @@ const PropertySelector = ({
                     schemaCopy.fields.length +
                     1
             )}`,
+            displayName: `${t(
+                'OATPropertyEditor.property'
+            )}_${String.fromCharCode(
+                ASCII_VALUE_BEFORE_LOWERCASE_ALPHABET +
+                    schemaCopy.fields.length +
```

Comment:
What happens if the length is really long? Like 100+? What chars will we get 🤔 ?